kernel/sql-federation/optimizer/src/main/java/org/apache/shardingsphere/sqlfederation/optimizer/metadata/schema/SQLFederationTable.java

```
@@ -124,4 +132,16 @@ public String toString() {
     public Statistic getStatistic() {
         return statistic;
     }
+    
+    @Override
+    public Collection<Object[]> getModifiableCollection() {
+        return null;
```

If this method does not support now, can we throw UnsupportedOperationExcetion?